File: announces-service/src/main/java/org/mts/announcesservice/remote_entities/Media.java
package org.mts.announcesservice.remote_entities;



import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Media {

    private Long id;

    // Mime type of the media (e.g. "image/png", "image/jpeg", "video/mp4")
    private String type;

    // Name of the media (e.g. "my-image.png")
    private String name;

    private String url;

    private String path;

    private Long size;
}

File: ui-angular/src/app/models/Media.ts
export interface Media {

  id : number,

  // Mime type of the media (e.g. "image/png", "image/jpeg", "video/mp4")
  type : string,

  // Name of the media (e.g. "my-image.png")
  name : string,

  url : string,

  path : string,

  size : number
}
